miner/scroll_worker.go

```diff
@@ -352,11 +352,12 @@ func (w *worker) mainLoop() {
 
 		select {
 		case <-w.startCh:
-			if err := w.checkHeadRowConsumption(); err != nil {
-				log.Error("failed to start head checkers", "err", err)
-				return
+			if w.isRunning() {
```

Comment:
Makes sense but what is the purpose of this? Will it break anything if follower nodes run this check on startup?

Reply:
I guess for follower nodes without ccc enabled, `asyncChecker` will be nil, and also nil row consumption in db. maybe the node would crash during starting.

Reply:
> Will it break anything if follower nodes run this check on startup?

Nope it wont. It is just confusing to see CCC logs without `—ccc`.